pkg/seccomp/default_linux.go

```diff
@@ -293,7 +294,6 @@ func DefaultProfile() *Seccomp {
 				"pidfd_send_signal",
 				"pipe",
 				"pipe2",
-				"pivot_root",
```

Comment:
isn't pivot_root used by container runtimes so this breaks the nested container case I guess?